molecule/default/verify.yml

```diff
@@ -245,3 +245,21 @@
       msg: |-
         'false' not found in output of 'dokku http-auth:report':
         {{ dokku_http_auth_off.stdout }}
+
+  # Testing dokku_checks
+  - name: Enabling the Zero Downtime deployment
```

Comment:
shouldn't we rather be testing disabling the checks then?

Reply:
hmm, I had tests for both cases, guess I removed it accidentally when rebasing